draft-ietf-oauth-rfc7523bis.xml

```diff
@@ -224,67 +205,6 @@
 	  </t>
 	</list>
       </t>
-      <t>
-	In Section 4 of <xref target="RFC7522"/> (Authorization Grant Example),
-	the sentence:
-	<list style="empty">
-	  <t>
-	    The intended audience of the Assertion is
-	    <spanx style="verb">https://saml-sp.example.net</spanx>,
-	    which is an identifier for a SAML Service Provider
-	    with which the authorization server identifies itself.
-	  </t>
-	</list>
-	is replaced by:
-	<list style="empty">
-	  <t>
-	    The intended audience of the Assertion is
-	    <spanx style="verb">https://authz.example.net</spanx>,
-	    which is the authorization server's issuer identifier.
-	  </t>
-	</list>
-      </t>
-      <figure title="Example SAML 2.0 Assertion" anchor="assertion">
-	<preamble>
-	  In the same section, the SAML 2.0 Assertion example is replaced by:
-	</preamble>
-	<artwork><![CDATA[
-  <Assertion IssueInstant="2025-07-17T00:53:34.619Z"
-    ID="ef1xsbZxPV2oqjd7HTLRLIBlBb7"
-    Version="2.0"
-    xmlns="urn:oasis:names:tc:SAML:2.0:assertion">
-   <Issuer>https://saml-idp.example.com</Issuer>
-   <ds:Signature xmlns:ds="http://www.w3.org/2000/09/xmldsig#">
-    [...omitted for brevity...]
-   </ds:Signature>
-   <Subject>
-    <NameID
-     Format="urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress">
-     brian@example.com
-    </NameID>
-    <SubjectConfirmation
-      Method="urn:oasis:names:tc:SAML:2.0:cm:bearer">
-     <SubjectConfirmationData
-       NotOnOrAfter="2025-07-17T00:58:34.619Z"
-       Recipient="https://authz.example.net/token.oauth2"/>
-     </SubjectConfirmation>
-    </Subject>
-    <Conditions>
-      <AudienceRestriction>
-        <Audience>https://authz.example.net</Audience>
-      </AudienceRestriction>
-    </Conditions>
-    <AuthnStatement AuthnInstant="2025-07-17T00:53:34.371Z">
-      <AuthnContext>
-        <AuthnContextClassRef>
-          urn:oasis:names:tc:SAML:2.0:ac:classes:X509
-        </AuthnContextClassRef>
-      </AuthnContext>
-    </AuthnStatement>
-  </Assertion>
-]]></artwork>
-      </figure>
-
```

Comment:
I disagree with deleting the updated example.  Having it is useful to readers.

Reply:
In what way is the inclusion of a lot of unnecessary intermediating/ugly/arcane XML content to replace a perfectly legitimate example useful to readers?

Reply:
> In my view, our spec should be replacing all the sections of the specs that it updates that we regard to be a bad practice with respect to the vulnerability. 

But there is no problem with using the SAML Entity ID, which is what hhttps://www.rfc-editor.org/rfc/rfc7522#section-4 does.